tests/common.py

```
@@ -0,0 +1,49 @@
+    if platform.system() == "Darwin":
+        default_target = default_target.replace(".so", ".dylib")
+    if release:
+        default_target = default_target.replace("debug", "release")
+    target = os.environ.get("TARGET", default_target)
+    r = Redis(port=port)
+    try:
+        r.ping()
+        client = FalkorDB(port=port)
+        g = client.select_graph("test")
+        return
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)